### DISCUSSION TITLE
RockPi instead...?

### DISCUSSION THREAD

**Discussion by deraragorn** [theyosh/TerrariumPI#786] [General]
Hey folks, hey yosh.

Is there a possibility to run TerrariumPi (via docker...?) on a RockPi?
As it is currently almost impossible to gather a Raspberry Pi 4...
If this is a sacrilege - don't strike me! 😵‍💫😁

Best wishes
Felix

**Comment by theyosh** [comment]
Hmm, not sure. I do depend on some PI hardware like the Raspberry Pi camera. So that could be a problem.

But, I think the easiest right now, is to try a docker image. https://theyosh.github.io/TerrariumPI/install/#docker

I am not sure how it will work, but I only tested this docker image on an ARM based OS. So not sure if it will work on a non ARM os as host It should be, but I do use a lot of hardware (GPIO pins, bluetooth, usb etc) and I do not know if all will stay to work with a non ARM os.
